src/funding/paypal/config.jsx

```diff
@@ -45,18 +45,21 @@ export function getPayPalConfig(): FundingSourceConfig {
 
     labelText: ({ content, label, period }) => {
       let text = `${FUNDING_BRAND_LABEL.PAYPAL}`;
+
       if (content && label === BUTTON_LABEL.INSTALLMENT) {
         if (period) {
-          text = content["label.installment.withPeriod"].replace(
-            "{period}",
-            String(period)
-          );
+          const rawLabel = content["label.installment.withPeriod"];
```

Comment:
fix from this slack discussion: https://paypal.slack.com/archives/CLKPA7686/p1719851055222409?thread_ts=1719526268.875739&cid=CLKPA7686